tests/unit_tests/layout/test_layout_array.py

```diff
@@ -272,11 +273,15 @@ def test_one_site(layout, site_label):
 
         layout.convert_coordinates()
         layout.print_telescope_list()
-        layout.export_telescope_list(crs_name="corsika")
+        _table = layout.export_telescope_list_table(crs_name="corsika")
+        _export_file = tmp_test_directory / "test_layout.ecsv"
+        _table.write(_export_file, format="ascii.ecsv", overwrite=True)
```

Comment:
in order to test whether it was written, I suggest opening the file and checking whether the table is the same as before.

Reply:
I think this is an integration test we need to do (and I have it on my list to implement).

Reply:
Still on this PR or in another one?

Reply:
No - we need to work on integration tests in our maintenance period (remember that we don't have good tests).